pepper-apis/src/main/java/org/broadinstitute/ddp/model/user/UserProfile.java

```diff
@@ -19,6 +19,7 @@
     private ZoneId timeZone;
     private Boolean doNotContact;
     private Boolean isDeceased;
+    private Boolean shouldSkipLanguagePopup;
```

Comment:
How about we name this `skipLanguagePopup`? We can keep `shouldDisplayLanguageChangePopup` in the study settings.